### PR TITLE
Correct macOS spelling

### PR DESCRIPTION
**Pull request by xla**
For Steve.

[wiki reference](https://en.wikipedia.org/wiki/MacOS)
[Apple reference](https://www.apple.com/macos/big-sur/)

### CODE REVIEW

**Comment by cloudhead** [comment]
You gotta rebuild

**Review by cloudhead — APPROVED** [review]

**Review by NunoAlexandre — DISMISSED** [review]

**Comment by brandonhaslegs** [comment]
I cannot believe I spelled this wrong! 🤔

**Review by brandonhaslegs — APPROVED** [review]

File: docs/downloads.html [hunk 1]
```diff
@@ -1,65 +0,0 @@
-<!doctype html>
```
Comment (NunoAlexandre) [inline docs/downloads.html:1]: Shouldn't this be rebuilt?

**Comment by xla** [comment]
I ran `make build`. :shrug: And then again.

**Review by NunoAlexandre — APPROVED** [review]

**Review by brandonhaslegs — APPROVED** [review]

**Comment by xla** [comment]
Done in 49d9a37

**Pull request closed without merging**